extensions/binding/org.eclipse.smarthome.binding.homematic/ESH-INF/thing/bridge.xml

```diff
@@ -7,6 +7,11 @@
 	<bridge-type id="bridge">
 		<label>Homematic Bridge</label>
 		<description>The Homematic bridge represents a Homematic gateway</description>
+		
+		<channels>
+			<channel typeId="DUTY_CYCLE_RATIO" id="DUTY_CYCLE_RATIO"/>
```

Comment:
There is no strict rule about naming conventions, but all-caps looks awkward compared to others. Most bindings use a lowercase+hypens, some use camelCase. Nevertheless, I understand that in the context of this binding, the dynamically generated channel names also are all-caps because that's what they are named like in the CCU. Therefore I think it makes sense to stay consistent. TL;DR: good as it is, no idea why I write this at all 😉